scripts/script.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const tableBody = document.getElementsByTagName('tbody')[0];
+
+async function gettingInfo(url)  {
```

Comment:
```suggestion
function gettingInfo(url)  {
```
async не потрібен, так як ти використовуєш `then`